core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheConfig.java

```diff
@@ -33,6 +33,12 @@ public class ChunkCacheConfig extends AbstractConfig {
             + "where \"-1\" represents infinite retention";
     private static final long DEFAULT_CACHE_RETENTION_MS = 600_000;
 
+    private static final String CACHE_PREFETCHING_SIZE_CONFIG = "prefetching.bytes";
```

Comment:
maybe `prefetch.size` based on number of chunks will be easier to configure. 
Even though an implementation detail, chunk size is a pretty core concept on the plugin  (e.g. config will be named `chunk.cache.prefetch.size`)